### PR TITLE
Set versions in Dockerfiles to 1

### PR DESCRIPTION
**Pull request by ekpyron**
This will hopefully trigger the builds of all docker images and will publish them to github packages in our repo.
If that works, we can have yet another PR that actually uses them in CircleCI.

### CODE REVIEW

**Review by bshastry — APPROVED** [review]
LGTM if all builder bots finish building :)

**Comment by ekpyron** [comment]
By the way: the best process for this is first letting the checks in this PR finish, then - before merging this PR - create another PR that actually uses the built docker images in CircleCI (the CI runs in this one will already have published the images, so they will be ready for use in another PR) - and only merge this one, when that other PR succeeds in its tests.
Because as long as we don't merge this, we can easily just modify the image builds and overwrite version 1 of the images :-).

**Comment by ekpyron** [comment]
Closing in favour of https://github.com/ethereum/solidity/pull/9430

**Pull request closed without merging**
